### DISCUSSION TITLE
noob question but pls help!

### DISCUSSION THREAD

**Discussion by eponine13** [ReagentX/imessage-exporter#109] [Q&A] [answered]
hello kind stranger(s)
i don't know the first thing about github or coding but would really like to use this tool to back up my imessages. i've managed to download and unzip the folder but i have no idea what to do next and how to use it. do i need to use a third party program to execute? would be so grateful for some help.

**Comment by ReagentX** [comment, marked as answer]
If you are trying to run one of the prebuilt releases, all you need to do is execute it in a terminal emulator. Assuming you saved the executable to your Downloads folder, it would look like this:

```zsh
% cd ~/Downloads
% chmod 755 ./imessage-exporter-aarch64-apple-darwin 
% ./imessage-exporter-aarch64-apple-darwin 
```

If you have GateKeeper enabled you will need to follow the steps [here](https://support.apple.com/en-us/HT202491).

Detail on the `chmod` step is located [here](https://support.apple.com/guide/terminal/make-a-file-executable-apdd100908f-06b3-4e63-8a87-32e71241bab4/mac).

**Reply by eponine13** [reply]
![Screenshot 2023-04-10 141143](https://user-images.githubusercontent.com/130330247/230824371-71d40493-2b93-471b-899f-81403669a8cc.png)
Sorry to be a pain - but this is what I see in my downloads, and this is what's inside the zip 
![image](https://user-images.githubusercontent.com/130330247/230824535-b4d449b4-e7dd-4549-8b1f-d20c42d39559.png)
I'm on a windows device so powershell is the terminal emulator (?) which shows up in the options (though I've downloaded cmder as well), and when I click open with powershell it gives me this 
![image](https://user-images.githubusercontent.com/130330247/230824679-6563759e-813d-4c9b-a305-a637fd8ab81a.png)
I'm a bit lost..

**Reply by ReagentX** [reply]
In your screenshot it looks like you downloaded the repository, not a release. Releases are located [here](https://github.com/ReagentX/imessage-exporter/releases).

I don't prebuild releases for Windows; if you want to run this code on that platform your best bet is to install it via Cargo, which will compile it for you. You can find details on using Cargo [here](https://doc.rust-lang.org/cargo/getting-started/installation.html).

More methods to install the app are listed in the [documentation](https://github.com/ReagentX/imessage-exporter/blob/develop/imessage-exporter/README.md).
